proof/refine/ARM/IpcCancel_R.thy

```diff
@@ -504,12 +504,12 @@ locale delete_one_conc = delete_one_conc_pre +
     "\<And>p. \<lbrace>invs'\<rbrace> cteDeleteOne p \<lbrace>\<lambda>rv. invs'\<rbrace>"
 
 locale delete_one = delete_one_conc + delete_one_abs +
-  assumes delete_one_corres:
+  assumes cteDeleteOne_corres:
     "corres dc (einvs and cte_wp_at can_fast_finalise ptr)
                (invs' and cte_at' (cte_map ptr))
           (cap_delete_one ptr) (cteDeleteOne (cte_map ptr))"
 
-lemma (in delete_one) reply_cancel_ipc_corres:
+lemma (in delete_one) cancelIPC_ReplyCap_corres:
```

Comment:
this should be `replyIPCCancel_corres`.  The concrete side is the RHS of the let binding for `replyIPCCancel`  inside `cancelIPC`.

(it would be nicer to call it `replyCancelIPC`, probably.)

Reply:
I'm happy to change it to `replyCancelIPC`